Cargo.toml

```diff
@@ -36,6 +36,9 @@ default = ["ort-download-binaries", "online"]
 online = ["hf-hub/online"]
 ort-download-binaries = ["ort/download-binaries"]
 ort-load-dynamic = ["ort/load-dynamic"]
+# This feature does not change any code, but is used to limit tests if
+# the user does not have `optimum-cli` or even python installed.
+optimum-cli = []
```

Comment:
How about installing it in the the GitHub CI action too?

Reply:
As per discussion below, this is not deemed feasible for now.